docs/astro/astro.config.mjs

```diff
@@ -31,10 +31,9 @@ export default defineConfig({
     },
     integrations: [
         starlight({
-            title: "1.9.2",
+            title: "Slint Docs",
```

Comment:
Please also adjust `.github/workflows/upgrade_version.yaml` to not touch this file anymore.